server/lib/settings.ts

```diff
@@ -191,6 +196,7 @@ interface AllSettings {
   sonarr: SonarrSettings[];
   public: PublicSettings;
   notifications: NotificationSettings;
+  jobs: JobSettings[];
```

Comment:
What about doing something like `Record<JobName, JobSettings>` instead? It will be easier to pull the job data later this way instead of doing find lookups. They can be empty objects by default maybe?

Reply:
Yep, that would be better. I'll make the change.